tests/test_admin.py

```diff
@@ -90,6 +90,27 @@ def test_filer_directory_listing_root_get(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.context['folder'].children.count(), 6)
 
+    def test_filer_directory_listing_performance(self):
+        images = 10
```

Comment:
Let's increase this number to some 100 images? That would really test the perf I think?

Reply:
Hi @vinitkumar ! Any number > 7 (no of queries expected) will ensure the there is no query for each file. i took 10 to save testing time and resources.